{"uuid":"134a0302-a40a-4623-b41f-a7af7260fd90","historyId":"cb2eb943d3996a09fffd36bca90828a0","testCaseId":"1a73765ccf7bf07909a1a976e1b39492","fullName":"src/test/resources/features/linkedList.feature:25","labels":[{"name":"tag","value":"smoke"},{"name":"host","value":"Blossoms"},{"name":"thread","value":"32744@Blossoms.main(1)"},{"name":"feature","value":"Linked List functionality"},{"name":"story","value":"validating Run function with valid code and invalid code"},{"name":"suite","value":"Linked List functionality"},{"name":"testClass","value":"validating Run function with valid code and invalid code"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/bloss/src/DsAlgoProject/src/test/resources/features/linkedList.feature"},{"name":"package","value":"src.test.resources.features.linkedList_feature.Linked List functionality"}],"links":[],"name":"validating Run function with valid code and invalid code","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Popup message did not match the expected error text. Actual:  expected [true] but found [false]","trace":"java.lang.AssertionError: Popup message did not match the expected error text. Actual:  expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat stepDefinitions.LinkedListSteps.the_user_gets_the_message(LinkedListSteps.java:69)\r\n\tat ✽.the user gets the message 'popup error message containing SyntaxError'(file:///C:/Users/bloss/src/DsAlgoProject/src/test/resources/features/linkedList.feature:22)\r\n"},"stage":"finished","steps":[{"name":"Given the user is logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733248457268,"stop":1733248459895},{"name":"Given the user is in the python editor of Introduction page","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733248459897,"stop":1733248460501},{"name":"When the user enters 'print(\\\"Hello\\\")abc' in the python editor and clicks Run","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733248460502,"stop":1733248460634},{"name":"Then the user gets the message 'popup error message containing SyntaxError'","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Popup message did not match the expected error text. Actual:  expected [true] but found [false]","trace":"java.lang.AssertionError: Popup message did not match the expected error text. Actual:  expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat stepDefinitions.LinkedListSteps.the_user_gets_the_message(LinkedListSteps.java:69)\r\n\tat ✽.the user gets the message 'popup error message containing SyntaxError'(file:///C:/Users/bloss/src/DsAlgoProject/src/test/resources/features/linkedList.feature:22)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733248460634,"stop":1733248460652}],"attachments":[],"parameters":[{"name":"code","value":"print(\\\"Hello\\\")abc"},{"name":"expectedOutcome","value":"popup error message containing SyntaxError"}],"start":1733248457217,"stop":1733248460826}